modelon/impact/client/experiment_definition.py

```diff
@@ -101,6 +101,32 @@ def __str__(self):
         return f"range({self.start_value},{self.end_value},{self.no_of_steps})"
 
 
+class Choices(Operator):
+    """
+    Choices operator class for parameterizing custom batch runs.
+
+    Parameters:
+
+        values --
+            Variable number of numerical arguments to sweep.
+
+    Examples::
+
+        from modelon.impact.client import Choices
+
+        fmu = model.compile().wait()
+        experiment_definition = fmu.new_experiment_definition(
+            custom_function).with_modifiers({'inertia1.J': 2,
+            'inertia2.J': Choices(0.1, 0.5)}, k=2, w=7)
```

Comment:
maybe just `({'inertia2.J': Choices(0.1, 0.5)})` to make the exmple easier to grasp?